public/pages/Snapshots/components/RestoreSnapshotFlyout/RestoreSnapshotFlyout.tsx

```diff
@@ -23,7 +23,7 @@ import { IndexService, SnapshotManagementService } from "../../../../services";
 import { RESTORE_OPTIONS } from "../../../../models/interfaces";
 import { getErrorMessage } from "../../../../utils/helpers";
 import { IndexItem } from "../../../../../models/interfaces";
-import { GetSnapshot } from "../../../../../server/models/interfaces";
+import { CatRepository, GetSnapshot, CatSnapshotIndex } from "../../../../../server/models/interfaces";
```

Comment:
Updates in this component are to enable user provided index settings to override or ignore via the IndexSettingsInput component